docs/en/latest/config.json

```diff
@@ -1,10 +1,14 @@
 {
-    "version": 0.4,
-    "sidebar": [
-      {
-        "type": "category",
-        "label": "Installation",
-        "items": ["apisix", "apisix-dashboard", "apisix-ingress-controller"]
-      }
-    ]
-  }
\ No newline at end of file
+  "version": 0.4,
+  "sidebar": [
+    {
+      "type": "category",
+      "label": "Installation",
+      "items": [
+        "apisix",
+        "apisix-dashboard",
+        "apisix-ingress-controller"
+      ]
+    }
+  ]
+}
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/112618947-acfa7880-8e61-11eb-993a-2fefb90ecae2.png)

Need a blank line at the ned of file

Reply:
Sorry, the auto prettier remove this, update the commit :)